schema.graphql

```diff
@@ -87,6 +87,20 @@ type Market @entity {
   exchangeRate: BigDecimal!
   "Collateral factor determining how much one can borrow"
   collateralFactor: BigDecimal!
+  "Stable interest debt token"
+  stableInterestDebtToken: Bytes
+  "Variable interest debt token"
+  variableInterestDebtToken: Bytes
+  "The next liquidity rate"
+  liquidityRate: BigInt
+  "The next stable borrow rate"
+  stableBorrowRate: BigInt
+  "The next variable borrow rate"
+  variableBorrowRate: BigInt
+  "The next liquidity Index"
+  liquidityIndex: BigInt
+  "The next variable Borrow Index"
+  variableBorrowIndex: BigInt
```

Comment:
A lot of these are AAVE specific fields. Which may have to be removed becasue they are not general enough. 

for now im okay keeping all of them. but they might need to be removed or renamed (for example `liquidityIndex` is poorly named by aave, it does not communicate what the variable represents)